### ISSUE TITLE
Prediction of Probabilistic Transient Stability Using Support Vector Machine. (arXiv:2111.11242v1 [eess.SY])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#13349]
Prediction of Probabilistic Transient Stability Using Support Vector Machine. (arXiv:2111.11242v1 [eess.SY])<br>
https://ift.tt/3xlraXp<br>
<p>Transient stability assessment is an integral part of dynamic security assessment of power systems. Traditional methods of transient stability assessment, such as time domain simulation approach and direct methods, are appropriate for offline studies and thus, cannot be applied for online transient stability prediction, which is a major requirement in modern power systems. This motivated the requirement to apply an artificial intelligence-based approach. In this regard, supervised machine learning is beneficial for predicting transient stability status, in the presence of uncertainties. Therefore, this paper examines the application of a binary support vector machine-based supervised machine learning, for predicting the transient stability status of a power system, considering uncertainties of various factors, such as load, faulted line, fault type, fault location and fault clearing time. The support vector machine is trained using a Gaussian Radial Basis function kernel and its hyperparameters are optimized using Bayesian optimization. Results obtained for the IEEE 14-bus test system demonstrated that the proposed method offers a fast technique for probabilistic transient stability status prediction, with an excellent accuracy. DIgSILENT PowerFactory and MATLAB was utilized for transient stability time-domain simulations (for obtaining training data for support vector machine) and for applying support vector machine, respectively.</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
